src/test/regress/pg_regress_multi.pl

```diff
@@ -56,6 +60,10 @@ ()
 my %functions = ();
 my %operators = ();
 my $valgrind = 0;
+my $valgrind_path = "valgrind";
+my $valgrind_log_file = "valgrind_test.log";
+my $pg_ctl_timeout = undef;
```

Comment:
`$valgrind` can be defined without defining `$pg_ctl_timeout`, or we may want to use `$pg_ctl_timeout` for some other purpose.